jax/_src/pjit.py

```diff
@@ -1131,7 +1139,7 @@ def _pjit_call_impl_python(
   compiled = _pjit_lower(
       jaxpr, in_shardings, out_shardings, resource_env,
       donated_invars, name, keep_unused, inline,
-      lowering_platform=None).compile()
+      lowering_parameters=mlir.LoweringParameters()).compile()
```

Comment:
Won't this break the compilation cache since you will be initializing a new object everytime?

This is the part I worry about the most.

Reply:
I assume that the compilation cache works by `__hash__`. That should be the same. Would there be a test that fails if I break the cache?

Reply:
Okay, i confirmed it locally that frozen=True works as expected.